get_github_coderabbit_review/coderabbit_fetcher/utils/memory_manager.py

```
@@ -0,0 +1,294 @@
+        try:
+            process = psutil.Process()
+            memory_info = process.memory_info()
+            system_memory = psutil.virtual_memory()
+
+            return MemoryStats(
+                used_mb=system_memory.used / 1024 / 1024,
+                available_mb=system_memory.available / 1024 / 1024,
+                percent_used=system_memory.percent,
+                process_mb=memory_info.rss / 1024 / 1024
+            )
+        except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)